packages/mobile/src/web3/reducer.ts

```diff
@@ -17,7 +16,6 @@ export interface State {
 }
 
 const initialState: State = {
-  isReady: false,
```

Comment:
I think since we don't persist this we don't need a redux-persist migration, but just a heads up for next time

Reply:
Yeah, as the migration tests didn't fail I figured. Is that how I'm supposed to check of this heuristic can fail?

Reply:
Yeah that would make sense to me - would defer to Rossy or Jean since they have worked with it more recently